### ISSUE TITLE
TypeError: Cannot read property 'aggregate' of undefined

### ISSUE THREAD

**Issue by Bojne** [dragonman225/notablog-starter#1]
Hi @dragonman225, as I run `npm run generate`, the following error popped out: 

```
I/notablog(20200202T171302.053): Fetch Site Metadata
TypeError: Cannot read property 'aggregate' of undefined
```

Not a JS guru - how should I resolve this issue?

**Comment by Bojne** [comment]
Also, I noticed the problems come from my Notion URLs:

- https://www.notion.so/yhhuang/0b21e6fc1cc94b61b925e91cab74cb3e?v=5859fa9c975c48c6bae16e658dbf84a7
- https://www.notion.so/0b21e6fc1cc94b61b925e91cab74cb3e?v=5859fa9c975c48c6bae16e658dbf84a7

Both result in the issue. Meanwhile, using your [original url](https://www.notion.so/b6fcf809ca5047b89f423948dce013a0?v=03ddc4d6130a47f8b68e74c9d0061de2) doesn't result in issue.

**Comment by dragonman225** [comment]
It's a known bug when using newer Notion table due to API changes. Currently there is no plan to patch it, but I'm working on a new version of Notablog, which will fix it.

**Comment by Bojne** [comment]
Oh! So maybe until they release the API, or you fresh up the new version, this will work. 

It's such a good idea to work this tool of on Notion as a CMS tho!

**Comment by Bojne** [comment]
Don't know if you're aware: [this similar project](https://notion-blog.now.sh/). It might be relevant!

**Comment by dragonman225** [comment]
> Oh! So maybe until they release the API, or you fresh up the new version, this will work.

Whether Notion makes their API public or not, it ([notablog](https://github.com/dragonman225/notablog)) needs an update to match the latest API.

> It's such a good idea to work this tool of on Notion as a CMS tho!

Yeah, Notion is a pretty good frontend for editing documents. Intuitive and expressive!

**Comment by dragonman225** [comment]
> Don't know if you're aware: [this similar project](https://notion-blog.now.sh/). It might be relevant!

I didn't notice this, but I have seen some [other similar projects](https://dragonman225.js.org/notion-resource.html#449da561-071b-4d68-8923-f767e2d82e7a). It seems that many people are interested in using Notion as a CMS, but there is no one best practice to do it. It's a relatively new field and people are exploring different methods.

**Comment by dragonman225** [comment]
Hi, I've published a beta version of Notablog, which fixed the `property 'aggregate' of undefined` bug.

Use the `develop` branch of this repository, you should be able try out Notablog with your Notion URL!

```
git clone https://github.com/dragonman225/notablog-starter.git -b develop
cd notablog-starter && npm install
```

**Comment by dragonman225** [comment]
This bug is fixed in [v0.4.0](https://github.com/dragonman225/notablog/tree/v0.4.0)
